pkg/services/notification/handler.go

```diff
@@ -107,10 +110,42 @@ func (s *Server) DeleteAddressList(ctx context.Context, req *pb.DeleteAddressLis
 	return &pb.DeleteAddressListResponse{}, nil
 }
 
-func (s *Server) SetServiceConfig(context.Context, *pb.ServiceConfig) (*pb.SetServiceConfigResponse, error) {
-	panic("implement me")
+func (s *Server) SetServiceConfig(ctx context.Context, req *pb.ServiceConfig) (*pb.SetServiceConfigResponse, error) {
+	protocol := req.GetEmailServiceConfig().GetProtocol().GetValue()
+	emailHost := req.GetEmailServiceConfig().GetEmailHost().GetValue()
+	port := req.GetEmailServiceConfig().GetPort().GetValue()
+	displayEmail := req.GetEmailServiceConfig().GetDisplayEmail().GetValue()
+	email := req.GetEmailServiceConfig().GetEmail().GetValue()
+	password := req.GetEmailServiceConfig().GetPassword().GetValue()
+	sslEnable := req.GetEmailServiceConfig().GetSslEnable().GetValue()
+
+	os.Setenv("NOTIFICATION_EMAIL_PROTOCOL", protocol)
+	os.Setenv("NOTIFICATION_EMAIL_EMAIL_HOST", emailHost)
+	os.Setenv("NOTIFICATION_EMAIL_PORT", port)
+	os.Setenv("NOTIFICATION_EMAIL_DISPLAY_EMAIL", displayEmail)
+	os.Setenv("NOTIFICATION_EMAIL_EMAIL", email)
+	os.Setenv("NOTIFICATION_EMAIL_PASSWORD", password)
+	os.Setenv("NOTIFICATION_EMAIL_SSL_ENABLE", strconv.FormatBool(sslEnable))
+
+	config.GetInstance().LoadConf()
+
+	return &pb.SetServiceConfigResponse{
+		IsSucc: pbutil.ToProtoBool(true),
+	}, nil
+
 }
 
-func (s *Server) GetServiceConfig(context.Context, *pb.GetServiceConfigRequest) (*pb.ServiceConfig, error) {
-	panic("implement me")
+func (s *Server) GetServiceConfig(ctx context.Context, req *pb.GetServiceConfigRequest) (*pb.ServiceConfig, error) {
+	serviceTypes := req.GetServiceType()
```

Comment:
# define these in constants.
const (
	ServiceTypeEmail = "email"
)

var ServiceTypes = []string{
	ServiceTypeEmail,
}

# here
if len(serviceTypes) == 0:
  serviceTypes=ServiceTypes